aiographite/protocol.py

```diff
@@ -0,0 +1,50 @@
+from typing import Tuple, List
+import pickle
+import struct
+
+
+class PlaintextProtocol:
+
+    def format_data(self, metric: str, value: int, timestamp: int) -> str:
```

Comment:
now that we have this, we can probably remove this method, and just always have format_data create a List[str].

Then move this logic to generate_message.